ScheduleAPI/Models/IDateNowRetriever.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace ScheduleAPI.Models
+{
+    public interface IDateNowRetriever
+    {
+        DateTime UtcNow { get; }
+    }
+}
```

Comment:
Can we add another Property that wraps the Now property of DateTime class.

Reply:
It's funny, I almost did that, but figured a given service will likely work in one or the other contexts. Might make sense for the implementation to contain both, but separate interfaces.